CHANGELOG.md

```diff
@@ -188,46 +181,27 @@ deprecated as they are still used in the `spanner::Connection` interface.)
 * fix(spanner): instantiate OptionsSpan objects in legacy admin calls ([#8440](https://github.com/googleapis/google-cloud-cpp/pull/8440))
 * feat(spanner): convert QueryOptions to Options in main client API ([#8430](https://github.com/googleapis/google-cloud-cpp/pull/8430))
 * fix(spanner): add Options save/restore to PartialResultSetSource ([#8355](https://github.com/googleapis/google-cloud-cpp/pull/8355))
-* feat(spanner): implement x-goog-user-project, part 2 ([#8332](https://github.com/googleapis/google-cloud-cpp/pull/8332))
 * feat(spanner): support x-goog-user-project ([#8316](https://github.com/googleapis/google-cloud-cpp/pull/8316))
```

Comment:
did you want to also link to #8332 ?

Reply:
I would proffer that we should either not mention it at all, or list it as a separate commit.